src/js/entities-service/tags.js

```diff
@@ -0,0 +1,23 @@
+import BaseEntity from 'js/base/entity-service';
+import { _Model, Model, Collection } from './entities/tags';
+
+let tagsCache;
+
+const Entity = BaseEntity.extend({
+  Entity: { _Model, Model, Collection },
+  radioRequests: {
+    'tags:model': 'getModel',
+    'tags:collection': 'getCollection',
+    'fetch:tags:collection': 'fetchTags',
+  },
+  fetchTags() {
+    if (tagsCache) return tagsCache;
```

Comment:
this makes it such that we're not expecting other tags to be made by other users during this user's session.  should be ok for now.. particularly since this is an rwell only feature